src/Gui/PythonConsolePy.cpp

```diff
@@ -78,9 +78,11 @@ Py::Object PythonStdout::repr()
 Py::Object PythonStdout::write(const Py::Tuple& args)
 {
     try {
-        Py::Object output(args[0]);
-        if (PyUnicode_Check(output.ptr())) {
-            PyObject* unicode = PyUnicode_AsEncodedString(output.ptr(), "utf-8", 0);
+        PyObject* output;
+        if (!PyArg_ParseTuple(args.ptr(), "O", &output))
+            throw Py::Exception();

```

Comment:
Could this maybe have some message attached to it? With these changes `sys.stderr.write()` returns nothing.